boards/arm/npcx7m6fb_evb/npcx7m6fb_evb_defconfig

```diff
@@ -49,6 +49,9 @@ CONFIG_ESPI=y
 # I2C Driver
 CONFIG_I2C=y
 
+# Sensor (Tachometer) Driver
+CONFIG_SENSOR=y
```

Comment:
This will enable the sensor in all applications, even simple ones like hello_world and blinky. Please see
[Board Porting](https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#porting-general-recommendations)

Reply:
Hi @MaureenHelm:
Thanks for your suggestion. Indeed, it will enable the tachometer in all applications using npcx7m6fb_evb board. I will remove it and the user should add this line in their application source if needed.

Reply:
Done.